admin/tabs/general/roles/RoleModel.ts

```diff
@@ -16,6 +16,13 @@ import {HoistRole, RoleMemberType, RoleServiceConfig} from './Types';
 export class RoleModel extends HoistModel {
     static PERSIST_WITH = {localStorageKey: 'xhAdminRolesState'};
 
+    static fmtDirectoryGroup(name?: string): string {
```

Comment:
don't think name is optional here

Reply:
Thanks - actually should be `string | null`, but I'll change to just `string`